jobs/validOwners.json

```diff
@@ -0,0 +1,29 @@
+[
+   "UNKNOWN",
+   "sig-api-machinery",
+   "sig-apps",
+   "sig-auth",
+   "sig-autoscaling",
+   "sig-aws",
+   "sig-big-data",
+   "sig-cli",
+   "sig-cluster-lifecycle",
+   "sig-cluster-ops",
+   "wg-contribex",
+   "sig-docs",
+   "sig-federation",
+   "sig-instrumentation",
+   "sig-network",
+   "sig-node",
+   "sig-onprem",
+   "sig-openstack",
+   "kubernetes-pm",
+   "kubernetes-release",
+   "sig-scale",
+   "sig-scheduling",
+   "sig-service-catalog",
+   "sig-storage",
+   "sig-testing",
+   "sig-ui",
+   "sig-windows"
+]
```

Comment:
Huh, never made sense to me why some people always put trailing newlines, so I finally looked it up and learned something about POSIX today: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

Sort of interesting that EOF doesn't officially end a line.